src/legacy/core_plugins/kibana/public/dashboard/dashboard_app_controller.tsx

```diff
@@ -251,17 +249,15 @@ export class DashboardAppController {
       !dashboardConfig.getHideWriteControls();
 
     $scope.updateQueryAndFetch = function({ query, dateRange }) {
-      if (dateRange) {
-        timefilter.setTime(dateRange);
-      }
-
-      const oldQuery = $scope.model.query;
-      if (_.isEqual(oldQuery, query)) {
+      if (_.isEqual($scope.model.query, query) && _.isEqual($scope.model.timeRange, dateRange)) {
```

Comment:
why was this changed ?

Reply:
I think I changed this while troubleshooting issues with the saved search embeddable. I can move this into a separate PR because I think it's unrelated enough.